querybook/server/lib/query_analysis/validation/base_query_validator.py

```diff
@@ -15,18 +20,21 @@ class QueryValidationSeverity(Enum):
 class QueryValidationResult(object):
     def __init__(
         self,
+        obj_type: QueryValidationResultObjectType,
```

Comment:
can you make this optional and default to QueryValidationResultObjectType.LINT? this makes sure other plugin still works as is